src/main/java/paimon/ui/UI.java

```diff
@@ -0,0 +1,50 @@
+package paimon.ui;
+
+import java.util.Scanner;
+
+/**
+ * UI class that interacts with the user.
+ * For example, it reads input from users. 
+ */
+public class UI {
```

Comment:
```suggestion
public class Ui {
```
Class name should be Ui and not UI: [[Relevant link in guide] ](https://se-education.org/guides/conventions/java/intermediate.html#naming) under the abbreviations section.

Reply:
Thank you!